internal/services/gateway/action/action.go

```diff
@@ -23,16 +23,18 @@ import (
 )
 
 type ActionHandler struct {
-	log               zerolog.Logger
-	sd                *common.TokenSigningData
-	configstoreClient *csclient.Client
-	runserviceClient  *rsclient.Client
-	agolaID           string
-	apiExposedURL     string
-	webExposedURL     string
+	log                                           zerolog.Logger
+	sd                                            *common.TokenSigningData
+	configstoreClient                             *csclient.Client
+	runserviceClient                              *rsclient.Client
+	agolaID                                       string
+	apiExposedURL                                 string
+	webExposedURL                                 string
+	organizationMemberAddingModeDirectEnabled     bool
+	organizationMemberAddingModeInvitationEnabled bool
```

Comment:
Use only one entry like before the change but don't import from config, declare a new enum here.